### PR TITLE
Add a popup when someone else throws something

### PR DESCRIPTION
**Pull request by DrSmugleaf**
## Media
image

## Requirements
- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

**Comment by SlamBamActionman** [comment]
Not sure I like this. It makes it harder to stealthily chuck bombs into rooms, throw explosive pens and makes the throwing knives even less appealing since you can track the thrower via the text pop-up. 

I could see this being useful for a downstream like RMC where throwing-capable characters are meant to all be on the same team, but for upstream it conflicts with the stealth aspect of certain roles.

**Comment by Pumkin69** [comment]
This PR does NOT seem like a good idea to merge into upstream.

**Comment by ElPidor** [comment]
BAD + SKILL ISSUE + DO NOT MERGE 
secoff dont care about throwing stuff like why non antag need to throw something? 
antag need to throw because have some things that are need to be thrown -> secoff see popup auga neuron activity -> stanbatoning 
its good as it is

**Review by Dutch-VanDerLinde — CHANGES_REQUESTED** [review]
this should use the identity of the entity who threw the item, believe this pop up will bypass identity

File: Content.Shared/Throwing/ThrowingSystem.cs
Comment (EmoGarbage404) [inline Content.Shared/Throwing/ThrowingSystem.cs:235]: need to use Identity.Entity() in `Content.Shared.IdentityManagement` for the user and uid so that it respects all the junk correctly.

File: Content.Shared/Throwing/ThrowingSystem.cs
Comment (EmoGarbage404) [inline Content.Shared/Throwing/ThrowingSystem.cs:236]: i'd probably say do a normal white popup instead of the scary red ones?

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]

**Comment by ps3moira** [comment]
This kinda kills any stealth if giant red text outs you

**Comment by DrSmugleaf** [comment]
i never expected this amount of powergamer malding over a basic feature that 13 has always had for very good reason
up to you i don't care what upstream does or does not merge though

**Comment by SlamBamActionman** [comment]
> i never expected this amount of powergamer malding over a basic feature that 13 has always had for very good reason up to you i don't care what upstream does or does not merge though

Please state the "very good reason" in the PR's "Why/Balance" section then (the one that was not included, in conflict with the Pull Request Guidelines), otherwise we are going to have to evaluate the feature based on what is being shown in relation to SS14's gameplay.

**Comment by Dutch-VanDerLinde** [comment]
why / balance : someone CHUCKING a whole ass bomb at someone stealthily is stupid

**Comment by ps3moira** [comment]
It's a good feature from SS13. There will eventually be more pop ups for actions so you can tell if someone's hitting someone, throwing something, reading or anything like that as it's kinda hard to tell at a certain points of the game what is happening. This is the starting point of that and should be welcomed

**Comment by SlamBamActionman** [comment]
Can't hide by throwing in/from the dark:
image
image

Disappearing into the crowd is no longer a strategy:
image

Text lingers after exiting FoV, so you can't throw it at someone unexpecting and then run away:
image

**Comment by BolloTea** [comment]
Epilepsy is the beginning.  If there is a pop-up window for every little thing, then your brain will simply explode trying to process all the information that pops up literally for a second and disappears forever.  At 13, at least it was in the chat, where you could pick up tons of text at any time and find what you need (plus there is a word search).  Here it’s done crookedly, poorly, and my God, seeing a pop-up window from any sneeze is annoying to say the least

**Comment by BolloTea** [comment]
> It's a good feature from SS13. There will eventually be more pop ups for actions so you can tell if someone's hitting someone, throwing something
>
Please - no.  The game already has a damage indicator to show that someone is hitting someone.  Extra information will be too overloading and unnecessary
Or do as in 13 - in the chat

**Comment by ps3moira** [comment]
> > It's a good feature from SS13. There will eventually be more pop ups for actions so you can tell if someone's hitting someone, throwing something
> 
> Please - no. The game already has a damage indicator to show that someone is hitting someone. Extra information will be too overloading and unnecessary Or do as in 13 - in the chat

A version of this that shows who throws or does what but in chat does seem better. I wonder if it's CCVar or easy to change to do this

**Comment by SlamBamActionman** [comment]
Hello! The PR was put up to a maintainer vote which concluded with the following result: Do Not Merge.

The reasons given were the following:
- Concerns about the stealth aspect of the game.
- Want to avoid too many pop-ups as it could lead to visual clutter and make people filter out pop-ups.

As such, this PR will be closed. Thank you for your contribution!

**Pull request closed without merging**
